mak/COPY

```diff
@@ -406,5 +406,6 @@ COPY=\
 	$(IMPDIR)\rt\array\equality.d \
 	$(IMPDIR)\rt\array\casting.d \
 	$(IMPDIR)\rt\array\capacity.d \
+	$(IMPDIR)\rt\array\concat.d \
 	\
```

Comment:
Minor nitpick: I guess to match the other modules, this should maybe be called rt.array.concatenation?

Reply:
:+1: